### PR TITLE
fix: table column styles leaking to other tables

### PR DESCRIPTION
**Pull request by kleinfreund**
**chore(meshes): address linter warning**

**fix(services): table column styles leaking to other tables**

**fix(meshes): table column styles leaking to other tables**

**fix(policies): table column styles leaking to other tables**

Signed-off-by: Philipp Rudloff <philipp.rudloff@konghq.com>

### CODE REVIEW

File: features/mesh/Index.feature [hunk 1]
```diff
@@ -2,7 +2,7 @@ Feature: mesh / index
   Background:
     Given the CSS selectors
       | Alias          | Selector                                     |
-      | items          | [data-testid='mesh-collection']              |
```
Comment (johncowen) [inline features/mesh/Index.feature:5]: Please could you leave this as collection

File: features/mesh/services/Index.feature [hunk 2]
```diff
@@ -2,7 +2,7 @@ Feature: mesh / services / index
   Background:
     Given the CSS selectors
       | Alias               | Selector                            |
-      | items               | [data-testid='service-collection']  |
```
Comment (johncowen) [inline features/mesh/services/Index.feature:5]: Please could you leave this as collection.

File: src/app/meshes/views/MeshListView.vue [hunk 3]
```diff
@@ -29,7 +29,8 @@
           <KCard>
             <template #body>
               <AppCollection
-                data-testid="mesh-collection"
```
Comment (johncowen) [inline src/app/meshes/views/MeshListView.vue:32]: See above

File: src/app/meshes/views/MeshListView.vue [hunk 4]
```diff
@@ -29,7 +29,8 @@
...
+                class="mesh-table"
+                data-testid="mesh-table"
```
Comment (johncowen) [inline src/app/meshes/views/MeshListView.vue:33]: See above

File: src/app/services/views/ServiceListView.vue [hunk 5]
```diff
@@ -19,7 +19,8 @@
         <KCard>
           <template #body>
             <AppCollection
-              data-testid="service-collection"
+              class="service-table"
+              data-testid="service-table"
```
Comment (johncowen) [inline src/app/services/views/ServiceListView.vue:23]: See above

**Review by johncowen — CHANGES_REQUESTED** [review]
Thanks for doing this! Couple of naming things that there is no need to change, in fact I would rather you didn't.

File: features/mesh/Index.feature (hunk 1 above)
Comment (kleinfreund) [reply features/mesh/Index.feature:5]: We already had *-table selectors in two other places so I stuck with that to not further have different patterns for this.

Comment (johncowen) [reply features/mesh/Index.feature:5]: Everything has moved to use `AppCollection` please could you use the term 'collection' for multiple things. You've actually changed them back to what we don't want here. Please could you use `collection` terminology, or at the very least leave where we are using `collection` terminology as it is.

File: src/app/meshes/views/MeshListView.vue [hunk 6]
```diff
@@ -29,7 +29,8 @@
           <KCard>
             <template #body>
               <AppCollection
-                data-testid="mesh-collection"
+                class="mesh-collections"
```
Comment (johncowen) [inline src/app/meshes/views/MeshListView.vue:32]: Singular please.

File: features/mesh/Index.feature (hunk 1 above)
Comment (kleinfreund) [reply features/mesh/Index.feature]: > to what we don't want here.

You. I was perfectly happy with calling those tables tables.

I changed those things now.

**Review by johncowen — APPROVED** [review]

**Pull request merged**